model/survey-response-logger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @module model/survey-response-logger
+ */
+
+const Sequelize = require('sequelize');
+
+/**
+ * Registers model with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * Each SurveyResponseLogger will be associated with a completed SurveyInstance.
+     * Each record represents the timestamps associated with when the question was answered.
```

Comment:
Yes, separate logic, but why separate database model?
This from a database stand point means we have two question responses, one with answers, another with timings.
That doesn't make sense.